store/src/main/java/org/apache/rocketmq/store/StoreStatsService.java

```diff
@@ -39,6 +43,12 @@
         "[<=0ms]", "[0~10ms]", "[10~50ms]", "[50~100ms]", "[100~200ms]", "[200~500ms]", "[500ms~1s]", "[1~2s]", "[2~3s]", "[3~4s]", "[4~5s]", "[5~10s]", "[10s~]",
     };
 
+    //The rule to define buckets
+    private static final Map<Integer, Integer> PUT_MESSAGE_ENTIRE_TIME_BUCKETS = new TreeMap<>();
+    //buckets
+    private TreeMap<Long/*bucket*/, AtomicLong/*times*/> buckets = new TreeMap<>();
```

Comment:
Consider using LongAdder to improve the performance

Reply:
OK. I will update it.